concurrent/common/Executors.java

```diff
@@ -58,7 +59,7 @@ private Executors(int parallelisation) {
         if (parallelisation <= 0) throw GraknException.of(ILLEGAL_ARGUMENT);
         // TODO: this is temporarily not used until we enable TransactionExecutor
         // serviceExecutorService = newFixedThreadPool(serviceThreadCount(parallelisation), threadFactory(GRAKN_CORE_SERVICE_THREAD_NAME));
```

Comment:
We should delete these comments.

Reply:
I think it's okay to keep `TODO` comments, but in this specific instance, the comment is no longer correct, so I'll remove it.